src/linter.ts

```diff
@@ -110,12 +113,16 @@ export const invoke = (
 ) => {
   process.chdir(cwd)
 
-  let cache = prettierCache.get(cwd)
+  const parsedOptions = parseArguments(args)
+  const prettierPath = parsedOptions.prettierPath
+  const cacheKey = `${cwd};${parsedOptions.prettierPath || ''}`
```

Comment:
Any reasoning behind including the prettier path in the cache key? I'm not against it, just curious on your thoughts behind it.

Reply:
The thought was to ensure there aren't cache collisions when people run `prettier_d_slim` in the same directory with different `--prettier-path` values. Wasn't entirely sure it mattered, but seemed like a good idea.

Reply:
👍 kinda figured!